network-protocols.md

```diff
@@ -81,7 +81,7 @@ type StorageDealProposal struct {
 	totalPrice TokenAmount
 
 	## Duration is how long the file should be stored for
-	Duration NumBlocks
+	Duration Int
```

Comment:
What unit is this in? seconds, blocks, hours, days?

Reply:
why is this off topic?? if you switch the type add a comment saying this is in blocks please